feature/agency/src/main/java/com/moneymong/moneymong/feature/agency/search/AgencySearchScreen.kt

```diff
@@ -238,25 +259,35 @@ private fun ContentViewWithoutAgencies(
         }
 
         is LoadState.NotLoading -> {
-            Column(
-                modifier = modifier,
-                verticalArrangement = Arrangement.spacedBy(
-                    space = 8.dp,
-                    alignment = Alignment.CenterVertically
-                ),
-                horizontalAlignment = Alignment.CenterHorizontally
-            ) {
-                Image(
-                    modifier = Modifier.size(size = 80.dp),
-                    painter = painterResource(id = R.drawable.img_agency),
-                    contentDescription = "agency image",
-                )
-                Text(
-                    text = "아직 등록된 소속이 없어요\n하단 버튼을 통해 등록해보세요",
-                    textAlign = TextAlign.Center,
-                    color = Gray08,
-                    style = Body4
+            Box(modifier = Modifier.fillMaxSize()) {
+                AgencyFeedbackItem(
+                    modifier = Modifier
+                        .align(Alignment.TopCenter)
+                        .padding(top = 10.dp),
+                    onClick = onClickFeedbackItem
                 )
+                Column(
+                    modifier = modifier
+                        .fillMaxSize()
+                        .align(Alignment.Center),
```

Comment:
```suggestion
                Column(
                    modifier = modifier
                        .fillMaxSize(),
```

이미 `fillMaxSize` 라서 `align` 이 필요가 없는 걸로 확인돼요!